src/mdx/expandVersionedUrl.js

```diff
@@ -0,0 +1,43 @@
+const tokens = [
```

Comment:
These tokens and rules are defined in a way that _I think_ could come from configuration instead, in case we decide to share this plugin with the rest of the world. I'm intending on reaching out to the docusaurus community to see if this kind of thing is useful for others.